src/FlightDisplay/GuidedAltitudeSlider.qml

```diff
@@ -95,6 +105,7 @@ Rectangle {
         maximumValue:       1
         zeroCentered:       true
         rotation:           180
+        groundIndicator:    _flying ? (1.0 - Math.pow(Math.max(0, Math.min(1, _distanceToGround / altLossRange)), 1.0/3.0)) * 0.5 : NaN
```

Comment:
This expression will fall out of sync with the altExp expression at some point, consider uniting the expression somehow

Reply:
Fixed, also remove unused _maxAlt/_minAlt while at it.